chore: change wording and move periods outside of links (#1703)
clerk/clerk-docs: docs/custom-flows/email-links.mdx

Your users will still be able to choose an alternative authentication (or verification) method even after they've clicked the email link they received in their inbox. Email links are simply the default authentication method for email address-based, passwordless authentication in Clerk.

> [!NOTE]

> > Looking for one-time code (OTP) authentication? See the [one-time code authentication](/docs/custom-flows/email-sms-otp) guide.

Reviewer: ```suggestion
> Looking for one-time code (OTP) authentication? See the [one-time code authentication guide](/docs/custom-flows/email-sms-otp).
```

## Before you start

> - You need to create a Clerk Application in your [Clerk Dashboard](https://dashboard.clerk.com/). For more information, check out our [Set up your application](/docs/quickstarts/setup-clerk) guide.

Reviewer: ```suggestion
- You need to create a Clerk Application in your [Clerk Dashboard](https://dashboard.clerk.com/). For more information, check out the [setup guide](/docs/quickstarts/setup-clerk).
```
